types/p-limit/index.d.ts

```diff
@@ -16,3 +16,4 @@ declare function limit<T, A>(cb: (a: A) => PromiseLike<T> | T, a: A): Promise<T>
 declare function limit<T>(cb: () => PromiseLike<T> | T): Promise<T>;
 
 declare function pLimit(concurrency: number): typeof limit;
+declare namespace pLimit {}
```

Comment:
Actually, you shouldn't do this. And [here's why](https://github.com/DefinitelyTyped/DefinitelyTyped#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports).

Refer to using `--allowSyntheticDefaultImports` or `--esModuleInterop` compiler options instead.

Reply:
Got it.  I'll go ahead and close this out then :) Thanks for the recommendation!